modules/prometheus/src/main/scala/com/evolutiongaming/smetrics/Prometheus.scala

```diff
@@ -11,6 +11,9 @@ trait Prometheus[F[_]] {
   def registry: CollectorRegistry[F]
 
   def write004: F[String]
+
+  // Initializes default hotspot metrics such as JVM threads, memory, GC, etc
+  def initDefaults: F[Unit]
```

Comment:
why do we need this at all here?

Reply:
For convenience. Basically if we don't add it, every team will have to initialise jvm metrics by hand(and remember to do it) like this Sync[F].delay(io.prometheus.client.hotspot.DefaultExports.register(collectorRegistry)). Also since we incapsulate registry here, we need to either give access to it or teams we'll have to initialize global registry(which probably always will work, but it seems counter intuitive since we encapsulate registry). I would even add it as a parameter to make method and initialise them on Prometheus[F] creation.